themes/Flex/templates/base.html

```diff
@@ -148,18 +148,18 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
 
     {% if MAIN_MENU %}
     <nav>
-      <a href="{{ SITEURL }}">{{ _('Home') }}</a>
+      <a href="{{ SITEURL }}" aria-label="Home page">{{ _('Home') }}</a>
 
       {% for title, link in MENUITEMS %}
-      <a href="{{ link }}">{{ title }}</a>
+      <a href="{{ link }}" aria-label="{{ title }}">{{ title }}</a>
       {% endfor %}
 
       {% if FEED_ALL_ATOM %}
-      <a href="{{ FEED_DOMAIN }}/{{ FEED_ALL_ATOM }}">{{ _('Atom') }}</a>
+      <a href="{{ FEED_DOMAIN }}/{{ FEED_ALL_ATOM }}" aria-label="Atom feed">{{ _('Atom') }}</a>
```

Comment:
These two are okay IFF the text content for them sucks - I don't see them on the live site so I can't confirm/deny what the content is.

Reply:
I think we may just not have these set to display!